### ISSUE TITLE
PSTest: -v & -of Command Line Arguments Ignored

### ISSUE THREAD

**Issue by bgribaudo** [microsoft/vscode-powerquery-sdk#132]
### Preflight Checklist

- [X] I have installed the [latest version of Power Query SDK](https://github.com/microsoft/vscode-powerquery-sdk/releases/latest).
- [X] I have checked existing resources, including the [common issues](https://learn.microsoft.com/en-us/power-query/commonissues) and the [release notes](https://github.com/microsoft/vscode-powerquery-sdk/releases).
- [X] I have searched for [similar issues](https://github.com/microsoft/vscode-powerquery-sdk/issues).

### Power Query SDK

0.1.5

### Platform

Windows

### Architecture

x64

### OS Version

Windows 10 Pro

### Bug Description

PQTest seems to ignore command line arguments `-v` and `-of`.

### Steps to Reproduce

Pass PSTest the arguments indicated in the "Actual Experience" section.

### Actual Experience

Output format/style/details are identical regardless of option specified. 
(Note: This holds true of argument syntax is changed from `-opt value` to `-opt=value`.)

## `-v` ignored:
`-v quiet`
````
PS> c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.1.5\.nuget\Microsoft.PowerQuery.SdkTools.2.110.2\tools\PQTest.exe run-test 
--extension "c:\Users\ben\Desktop\TestConnector\BenTest\bin\AnyCPU\Debug\BenTest.mez" 
--queryFile "c:\Users\ben\Desktop\TestConnector\BenTest\BenTest.query.pq" 
-v quiet

[{"ActivityId":"9a63b416-d1c1-4625-b323-102752417d0a","DataSourceAnalysis":[{"Kind":"DataSource","FunctionName":"BenTest.Configuration","DataSource":{"kind":"BenTest","path":"BenTest"},"DataSourceReference":{"protocol":"x-datasource","address":{"kind":"BenTest","path":"BenTest"},"authentication":null,"query":null},"ModuleName":null,"ModuleVersion":null}],"Details":"let\r\n    Result = BenTest.Configuration()\r\nin\r\n    Result","EndTime":"2022-09-28T13:35:36.5747031+00:00","Method":"PQTest.RunTest","Name":"BenTest.query.pq","StartTime":"2022-09-28T13:35:35.4607274+00:00","Output":[],"RowCount":0,"Status":"Passed","TraceFileLocation":null,"Type":"PQTest.Expression"}]
````

`-v detailed`
````
PS> c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.1.5\.nuget\Microsoft.PowerQuery.SdkTools.2.110.2\tools\PQTest.exe run-test 
--extension "c:\Users\ben\Desktop\TestConnector\BenTest\bin\AnyCPU\Debug\BenTest.mez" 
--queryFile "c:\Users\ben\Desktop\TestConnector\BenTest\BenTest.query.pq" 
-v detailed

[{"ActivityId":"e8672124-cede-4417-abb4-3495c181c47c","DataSourceAnalysis":[{"Kind":"DataSource","FunctionName":"BenTest.Configuration","DataSource":{"kind":"BenTest","path":"BenTest"},"DataSourceReference":{"protocol":"x-datasource","address":{"kind":"BenTest","path":"BenTest"},"authentication":null,"query":null},"ModuleName":null,"ModuleVersion":null}],"Details":"let\r\n    Result = BenTest.Configuration()\r\nin\r\n    Result","EndTime":"2022-09-28T13:35:43.3867876+00:00","Method":"PQTest.RunTest","Name":"BenTest.query.pq","StartTime":"2022-09-28T13:35:42.2707326+00:00","Output":[],"RowCount":0,"Status":"Passed","TraceFileLocation":null,"Type":"PQTest.Expression"}]
````

## `-of` Ignored
`-of XUnit`
````
PS> c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.1.5\.nuget\Microsoft.PowerQuery.SdkTools.2.110.2\tools\PQTest.exe run-test --extension "c:\Users\ben\Desktop\TestConnector\BenTest\bin\AnyCPU\Debug\BenTest.mez" 
--queryFile "c:\Users\ben\Desktop\TestConnector\BenTest\BenTest.query.pq" 
-of XUnit

[{"ActivityId":"3eec45a8-5024-4500-ab1d-7f6d4a922ff4","DataSourceAnalysis":[{"Kind":"DataSource","FunctionName":"BenTest.Configuration","DataSource":{"kind":"BenTest","path":"BenTest"},"DataSourceReference":{"protocol":"x-datasource","address":{"kind":"BenTest","path":"BenTest"},"authentication":null,"query":null},"ModuleName":null,"ModuleVersion":null}],"Details":"let\r\n    Result = BenTest.Configuration()\r\nin\r\n    Result","EndTime":"2022-09-28T13:36:05.9674521+00:00","Method":"PQTest.RunTest","Name":"BenTest.query.pq","StartTime":"2022-09-28T13:36:04.8577669+00:00","Output":[],"RowCount":0,"Status":"Passed","TraceFileLocation":null,"Type":"PQTest.Expression"}]
````

`-of ADO`
````
PS> c:\Users\ben\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.1.5\.nuget\Microsoft.PowerQuery.SdkTools.2.110.2\tools\PQTest.exe run-test 
--extension "c:\Users\ben\Desktop\TestConnector\BenTest\bin\AnyCPU\Debug\BenTest.mez" 
--queryFile "c:\Users\ben\Desktop\TestConnector\BenTest\BenTest.query.pq" 
-of ADO

[{"ActivityId":"d4dc07b8-cca2-4ae4-95fe-1c00283679a0","DataSourceAnalysis":[{"Kind":"DataSource","FunctionName":"BenTest.Configuration","DataSource":{"kind":"BenTest","path":"BenTest"},"DataSourceReference":{"protocol":"x-datasource","address":{"kind":"BenTest","path":"BenTest"},"authentication":null,"query":null},"ModuleName":null,"ModuleVersion":null}],"Details":"let\r\n    Result = BenTest.Configuration()\r\nin\r\n    Result","EndTime":"2022-09-28T13:36:11.7247293+00:00","Method":"PQTest.RunTest","Name":"BenTest.query.pq","StartTime":"2022-09-28T13:36:10.6115853+00:00","Output":[],"RowCount":0,"Status":"Passed","TraceFileLocation":null,"Type":"PQTest.Expression"}]
````

### Expected Experience

Each argument should affect the output produced. E.g. `-v detailed` should add verbose details to the output and `-v XUnit` should  change the formatting from JSON (the default) to XUnit-style.

**Comment by mattmasson** [comment]
These options were part of the unfinished unit testing work. I'll remove them until they are ready to be used.

**Comment by mattmasson** [comment]
This is resolved with the latest SDK Tools update - 2.110.3.

**Comment by bgribaudo** [comment]
Thanks, @mattmasson!
